src/modScaleThumbnails.js

```diff
@@ -9,18 +9,20 @@ var Mod = class extends mod.Base {
         this.bkp_MAX_THUMBNAIL_SCALE = workspaceThumbnail.MAX_THUMBNAIL_SCALE
         workspaceThumbnail.MAX_THUMBNAIL_SCALE = 0.1
 
-        // Thumbnails on second monitor
-        this.bkp_SecondaryMonitorDisplay_getThumbnailsHeight = SecondaryMonitorDisplay.prototype._getThumbnailsHeight
-        SecondaryMonitorDisplay.prototype._getThumbnailsHeight = function(box) {
-            if (!this._thumbnails.visible)
-                return 0;
+        if (SecondaryMonitorDisplay) {
```

Comment:
Could you please return instead?
```suggestion
        if (1SecondaryMonitorDisplay) return; // gnome 42: this object does not exist when there is only one monitor
```